packages/sn-filter-pane/api-spec/spec.json

```
@@ -0,0 +1,109 @@
+{
+  "scriptappy": "1.1.0",
+  "info": {
+    "name": "@nebula.js/sn-filter-pane:properties",
+    "description": "Filter pane generic object definition",
+    "version": "0.8.3",
+    "license": "MIT",
+    "stability": "experimental",
+    "x-qlik-visibility": "public"
+  },
+  "entries": {
+    "properties": {
+      "kind": "namespace",
+      "entries": {
+        "footnote": {
+          "description": "Visualization footnote.",
+          "optional": true,
+          "defaultValue": "",
+          "kind": "union",
+          "items": [
+            {
+              "type": "string"
+            },
+            {
+              "type": "StringExpression"
+            }
+          ]
+        },
+        "qChildListDef": {
+          "description": "Child object definition",
+          "defaultValue": "{qDef:{qListObjectDef:{}}}",
+          "type": "#/definitions/ChildListDef"
+        },
+        "showDetails": {
+          "description": "Show visualization details toggle",
+          "optional": true,
+          "defaultValue": true,
+          "type": "boolean"
+        },
```

```suggestion
        "showDetails": {
          "description": "Show visualization details toggle",
          "optional": true,
          "defaultValue": true,
          "type": "boolean"
        },
```

This should actually not be included here, since this doesn't work outside of the sense-client.

---

we do have it in most of our other nebula charts. (but maybe they also are wrong)

---

Okey, yeah, I think it might be better not to include it to avoid confusion for people rendering using nebula on a mashup, until we have support for "details" in nebula (if we ever will...)

---

I don't know if I agree.
the spec is also for how to create object used inside client.
maybe a comment that is only is relevant for that use case would be a better solution.

---

That's a way to solve it as well, and that's probably better.
So yes, we can have it here, but mention that it only works inside Qlik Sense.